ai-machine-learning/thoth-thamos/setup.md

```diff
@@ -0,0 +1,28 @@
+Before start using the Thamos let's setup the environment.
+
+Next we could go ahead and install `Thamos CLI` -
+
+``pip3 install thamos``{{execute}}
+
+
+Now that all that is done, let's check what Thamos offer-
+
+``thamos --help``{{execute}}
+
+You should see something like this -
+![Thamos help](https://raw.githubusercontent.com/saisankargochhayat/katacoda-scenarios/master/thamos-cli/assets/thamos_help.png)
+
+Bravo if you made it till here! We are almost done with the setup. 
+Last step is cloning out demo app - [Link](https://github.com/thoth-station/s2i-example)
```

Comment:
is it supposed to be `our demo app` maybe?

Reply:
thanks